deploy/curve-yield-strategy-logic.ts

```diff
@@ -13,12 +16,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const logicLib = await get('LogicLibrary');
   const swapManagerLib = await get('SwapManagerLibrary');
 
-  await deploy('CurveYieldStrategyLogic', {
+  await (deploy as unknown as DeployExtended<CurveYieldStrategy__factory>)('CurveYieldStrategyLogic', {
     contract: 'CurveYieldStrategy',
     libraries: {
       SwapManager: swapManagerLib.address,
       Logic: logicLib.address,
     },
+    args: [(await get('SwapSimulator')).address],
```

Comment:
would this give the correct address (i.e latest one with changes)? does any dependency or network-info need to be update?

Reply:
Yeah I think I need to make sure @ragetrade/core package is correct, it pulls the SwapSimulator address from there.